src/Common/Admin/Help_Hub/Hub.php

```diff
@@ -0,0 +1,537 @@
+			Tribe__Main::instance(),
+			'help-hub-iframe-js',
+			'admin/help-hub-iframe.js',
+			null,
+			'wp_enqueue_scripts',
+			[
+				'localize' => [
+					'name' => 'helpHubSettings',
+					'data' => [
+						'docsbot_key'    => $this->config->get( 'TEC_HELP_HUB_CHAT_DOCSBOT_SUPPORT_KEY' ),
+						'zendeskChatKey' => $this->config->get( 'TEC_HELP_HUB_CHAT_ZENDESK_CHAT_KEY' ),
+					],
+				],
+			]
+		);
+
+		self::dequeue_theme_styles();
+	}
+
+	/**
+	 * Removes theme-related styles to avoid iframe conflicts.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	protected static function dequeue_theme_styles(): void {
+		global $wp_styles;
+		$theme_directory = get_template_directory_uri();
+
+		foreach ( $wp_styles->queue as $handle ) {
+			$src = $wp_styles->registered[ $handle ]->src;
+			if ( strpos( $src, $theme_directory ) !== false ) {
+				wp_dequeue_style( $handle );
+			}
+		}
+	}
+
+	/**
+	 * Generates HTML for the admin notice.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The HTML for the admin notice.
+	 */
+	private function generate_notice_html(): string {
+		$notice_slug    = 'tec-common-help-chatbot-notice';
+		$notice_content = sprintf(
+			// translators: Placeholders are for the `a` tag that displays a link.
+			_x(
+				'To find the answer to all your questions use the %1$sTEC Chatbot%2$s',
+				'The callout notice to try the chatbot with a link to the page',
+				'tribe-common'
+			),
+			'<a data-tab-target="tec-help-tab" href="#">',
+			'</a>'
+		);
+
+		$notice_admin = ( new AdminNotice( $notice_slug, "<p>$notice_content</p>" ) )
+			->urgency( 'info' )
+			->inline()
+			->dismissible( false )
+			->withWrapper();
+
+		return AdminNotices::render( $notice_admin, false );
+	}
+
+	/**
+	 * Renders the specified template with provided variables.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $template_name The template file name (without .php).
+	 * @param array  $extra_values  Extra values to pass to the template.
+	 *
+	 * @return void
+	 */
+	private function render_template( string $template_name, array $extra_values = [] ): void {
+		$main     = Tribe__Main::instance();
+		$template = $this->template;
+
+		$template_values = array_merge(
+			[
+				'main'          => $main,
+				'status_values' => $this->get_status_values(),
+				'keys'          => $this->get_chat_keys(),
+				'icons'         => $this->data->get_icon_urls( $main ),
+				'links'         => self::get_links(),
+			],
+			$extra_values
+		);
+
+		$template->set_values( $template_values );
+		$template->set_template_origin( $main );
+		$template->set_template_folder( 'src/admin-views' );
+		$template->set_template_context_extract( true );
+		$template->set_template_folder_lookup( false );
+		$template->template( $template_name );
+	}
+
+	/**
+	 * Retrieves the opt-in status and license validity.
+	 *
+	 * @since TBD
+	 *
+	 * @return array Associative array with opt-in and license status.
+	 */
+	protected function get_status_values(): array {
```

Comment:
This is redundant - it's just a call to another function that does the exact same thing.